atc/db/team_factory.go

```diff
@@ -112,7 +112,7 @@ func (factory *teamFactory) FindTeam(teamName string) (Team, bool, error) {
 func (factory *teamFactory) GetTeams() ([]Team, error) {
 	rows, err := psql.Select("id, name, admin, auth").
 		From("teams").
-		OrderBy("id ASC").
+		OrderBy("name ASC").
```

Comment:
Does this affect the order that the teams are displayed in the UI? It seems to be used by the `ListTeams` endpoint, does that endpoint get used to display the ordering?

I tried removing this change and spun up a server locally and I don't see it making any difference. Maybe I'm missing something about where this endpoint is being used?

Reply:
`GetTeams` is called by accessor to generate auth token, thus order is not an issue. But I think it's better to keep consistent for all places where needs to order teams, otherwise code reader might be confused why here order by id and there order by name. WDYT?

Reply:
@evanchaoli Ahh yeah I see it's used by the `fly teams` command. Then that does make sense for it to also be ordered by name. I think if it was just used by the accessor to generate an auth token, I would've been slightly unsure about having it be ordered by name? Just because I would rather than the queries be completely representative of what they are trying to fetch (if ordering by name matters or not) over being consistent. But this method is being used by a fly command to display the teams so I am 100% for having it be ordered by name too!